src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -43,13 +37,16 @@ public static void main(String[] args) throws IOException {
     @SuppressWarnings("unchecked")
     protected void doOperation() {
 
-        String q = "SELECT FROM " + CourseStudent.class.getName();
-        List<CourseStudent> allStudents = (List<CourseStudent>) PM.newQuery(q).execute();
+        String queryString = "SELECT FROM " + CourseStudent.class.getName();
+        List<CourseStudent> allStudents = (List<CourseStudent>) PM.newQuery(queryString).execute();
 
-        q = "SELECT FROM " + Instructor.class.getName();
-        List<Instructor> allInstructors = (List<Instructor>) PM.newQuery(q).execute();
+        queryString = "SELECT FROM " + Instructor.class.getName();
+        List<Instructor> allInstructors = (List<Instructor>) PM.newQuery(queryString).execute();
 
-        StatsBundle statsBundle = generateStatsPerInstitute(allStudents, allInstructors);
+        queryString = "SELECT FROM " + Account.class.getName();
+        List<Account> allAccounts = (List<Account>) PM.newQuery(queryString).execute();
+
+        StatsBundle statsBundle = generateStatsPerInstitute(allStudents, allInstructors, allAccounts);
```

Comment:
@damithc Is this something we need to do? I remember you mentioning something against it.